arm-datalake-analytics/catalog/2016-11-01/swagger/catalog.json

```diff
@@ -2767,11 +3343,12 @@
           "description": "the assembly file type.",
           "enum": [
             "Assembly",
-            "Resource"
+            "Resource",
+            "Nodeploy"
           ],
           "x-ms-enum": {
             "name": "FileType",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
is this an intentional change? I believe this is a potential breaking change, I don't see other breaking changes in this PR, therefore the question.